utility/service/service_test.go

```diff
@@ -11,25 +15,53 @@ import (
 	coreTypes "github.com/pokt-network/pocket/shared/core/types"
 	"github.com/pokt-network/pocket/shared/modules"
 	mockModules "github.com/pokt-network/pocket/shared/modules/mocks"
+	typesUtil "github.com/pokt-network/pocket/utility/types"
 )
 
-var testServicer1 = &coreTypes.Actor{
-	ActorType: coreTypes.ActorType_ACTOR_TYPE_SERVICER,
-	Address:   "a3d9ea9d9ad9c58bb96ec41340f83cb2cabb6496",
-	PublicKey: "a6cd0a304c38d76271f74dd3c90325144425d904ef1b9a6fbab9b201d75a998b",
-	Chains:    []string{"0021"},
-}
+const (
+	testAppsTokensMultiplier = int(2)
+	testCurrentHeight        = int64(9)
+)
+
+var (
+	testServicer1 = &coreTypes.Actor{
+		ActorType: coreTypes.ActorType_ACTOR_TYPE_SERVICER,
+		Address:   "a3d9ea9d9ad9c58bb96ec41340f83cb2cabb6496",
+		PublicKey: "a6cd0a304c38d76271f74dd3c90325144425d904ef1b9a6fbab9b201d75a998b",
+		Chains:    []string{"0021"},
+	}
+
+	testApp1 = &coreTypes.Actor{
+		Address:      "98a792b7aca673620132ef01f50e62caa58eca83",
+		PublicKey:    "b5cd0a304c38d76271f74dd3c90325144425d904ef1b9a6fbab9b201d86b009c",
+		StakedAmount: "1000",
+	}
+
+	testChainConfig1 = &configs.ChainConfig{
+		Url:                 "http://chain-url.pokt.network",
+		UserAgent:           "user-agent-1",
+		TimeoutMilliseconds: 1234,
+		BasicAuth: &configs.BasicAuth{
+			UserName: "user1",
+			Password: "password1",
+		},
+	}
+)
 
 func TestAdmitRelay(t *testing.T) {
-	currentHeight := uint64(9)
+	const (
+		currentSessionNumber = 2
+	)
+
 	testCases := []struct {
-		name     string
-		relay    *coreTypes.Relay
-		expected error
+		name       string
+		usedTokens int64
```

Comment:
s/usedTokens/usedSessionTokens

Is this `total` or `perRelay`?

Reply:
Thank you for the review. Fixed.